src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1,4 +1,8 @@
 Unable\ to\ monitor\ file\ changes.\ Please\ close\ files\ and\ processes\ and\ restart.\ You\ may\ encounter\ errors\ if\ you\ continue\ with\ this\ session.=Unable to monitor file changes. Please close files and processes and restart. You may encounter errors if you continue with this session.
+
+Could\ not\ move\ file.\ Please\ close\ all\ the\ attached\ files\ and\ retry.=Could not move file. Please close all the attached files and retry.
```

Comment:
Please include the filename here. Example: If have 100 files attached, 2 are opened, two cause issues, 98 not. Does the user know which file to close?!